Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "cosm-tome"
-version = "0.1.1"
+name = "cosm-tome-wasm-deploy-fork"
+version = "0.1.0"
```

Comment:
Can you the revert package name and version changes?

Not sure if this was an accidental merge from one of your test branches or not.

Reply:
Oh wow whoops. Yes that was accidental. I created a temporary fork so I could publish my project on crates.io and forgot to make a separate branch for that. Should be fixed now.